### DISCUSSION TITLE
More real estate on the screen(widescreen support)

### DISCUSSION THREAD

**Discussion by Syphorlate** [ActivityWatch/activitywatch#913] [Ideas]
Would be much better if AW would be using more of the available real estate on the screen to be able to dock more visualizations. Also visualizations that are large in height such as the score window will block a lot of area where you can't dock anything then. Would be great if this could be improved!

**Comment by ErikBjare** [comment]
I agree!

We made the change for the Timeline view a while back, which was a big improvement imo, so have been considering the same for the Activity view, but would require a significant overhaul of the layout system, so might be a while...

**Reply by Syphorlate** [reply]
Thank you very much for your consideration and for having brought us such a great tool!
